tests/test_flake8_trio.py

```diff
@@ -40,9 +40,19 @@ def trim_messages(messages: Iterable[Error]):
         errors = tuple(plugin.run())
 
         # start with a check with trimmed errors that will make for smaller diff messages
-        trim_errors = trim_messages(plugin.run())
+        trim_errors = trim_messages(errors)
         trim_expected = trim_messages(expected)
-        self.assertTupleEqual(trim_errors, trim_expected)
+
+        #
```

Comment:
missing comment

Reply:
This function is getting a makeover in #19 (which has merged this branch), so I'd leave it for the moment and avoid a merge conflict.